src/styles/global.scss

```
@@ -0,0 +1,13 @@
+@import "./variables.scss";
+@import "./mixins.scss";
+@import "./card";
+@import "./stars";
```

Please be consistent and keep or in this case remove the extension for all scss imports
```suggestion
@import "./variables";
@import "./mixins";
@import "./card";
@import "./stars";
```

---

Ok) Thanks)